README.md

```diff
@@ -1,149 +1,181 @@
 # Deferred
-
-This is an implementation of [OCaml's Deferred](https://ocaml.janestreet.com/ocaml-core/111.25.00/doc/async_kernel/#Deferred) for Swift.
-
-## Overview
-
-`Deferred` is designed for supporting asynchronous programming. An instance of
-`Deferred` represents a value that will be available at some point in the
-future. Deferred objects can trivially replace completion blocks (see
-[Running Closures Upon Fulfillment](#upon)), but also enable some higher level,
-powerful composition techniques.
-
-All properties and methods on an instance of `Deferred` can safely be called from
-multiple threads simultaneously; a lock is used internally for synchronization.
-Obviously this does not guarantee thread-safety of the contained result (which
-`Deferred` knows nothing about).
-
-An instance of `Deferred` can only be filled once. It is a programmer error to
-fill an already-filled `Deferred`, and this will result in a runtime trap. (The
-method `fillIfUnfilled` is available for conditional filling.)
-
-## Usage - Producer
+## Vital Statistics
+- What: Lets you work with values that haven't been determined yet,
+        like an array that's coming (one day!) from a web service call.
```

Comment:
That sort of rewriting goes great with a small PR. :)

El sábado, 15 de agosto de 2015, Step Christopher <notifications@github.com>
escribió:

> In README.md
> <https://github.com/bignerdranch/Deferred/pull/16#discussion_r37140977>:
>
> > -[Running Closures Upon Fulfillment](#upon)), but also enable some higher level,
> > -powerful composition techniques.
> > -
> > -All properties and methods on an instance of `Deferred` can safely be called from
> > -multiple threads simultaneously; a lock is used internally for synchronization.
> > -Obviously this does not guarantee thread-safety of the contained result (which
> > -`Deferred` knows nothing about).
> > -
> > -An instance of `Deferred` can only be filled once. It is a programmer error to
> > -fill an already-filled `Deferred`, and this will result in a runtime trap. (The
> > -method `fillIfUnfilled` is available for conditional filling.)
> > -
> > -## Usage - Producer
> > +## Vital Statistics
> > +- What: Lets you work with values that haven't been determined yet,
> > +        like an array that's coming (one day!) from a web service call.
>
> one day! -> eventually. It is funny but only if you already know what is
> meant. Better to be clearer here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bignerdranch/Deferred/pull/16/files#r37140977>.
>


-- 
--
Jeremy W. Sherman <jeremy@bignerdranch.com>